README.dev.md

```diff
@@ -134,30 +222,39 @@ sudo apt-get install closure-compiler libfreetype6-dev libffi-dev \
     postgresql-server-dev-all libmemcached-dev python-dev \
     hunspell-en-us nodejs nodejs-legacy npm git yui-compressor \
     puppet gettext
+``` 
 
-# If on 12.04 or wheezy:
+If you are running Ubuntu 12.04 or Debian Wheezy, run the following 
+commands in the terminal:
+```
 sudo apt-get install postgresql-9.1
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.1-tsearch-extras_0.1.2_amd64.deb
 sudo dpkg -i postgresql-9.1-tsearch-extras_0.1.2_amd64.deb
+```
 
-# If on 14.04:
+If you are running Ubuntu 14.04, run the following commands
+in the terminal:
+```
 sudo apt-get install postgresql-9.3
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
 sudo dpkg -i postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
+```
 
-# If on 15.04 or jessie:
+If you are running Ubuntu 15.04 or Debian Jessie, run the 
+following commands in the terminal:
+```
 sudo apt-get install postgresql-9.4
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.4-tsearch-extras_0.1_amd64.deb
 sudo dpkg -i postgresql-9.4-tsearch-extras_0.1_amd64.deb
 ```
-
 Now continue with the "All systems" instructions below.
 
-### On Fedora 22 (experimental):
+If you are running on Fedora 22 (experimental):
```

Comment:
Why did you make these no longer headings?  The previous model had the advantage that it's easy to skim for your distro and/or link to the instructions for a particular distro..